apps/storybook-react-native/tsconfig.build.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
non-blocking: Can be removed as this wont be a published package

Reply:
can you clarify?